docs/deploy-local.md

```diff
@@ -49,19 +48,18 @@ In addition to the Kubernetes version requirement in [Deploying CF for K8s](depl
 
 1. Follow the instructions in [Deploying CF for K8s](deploy.md).
 
-   - Include the [remove-resource-requirements.yml](../config-optional/remove-resource-requirements.yml) and
-     [remove-ingressgateway-service.yml](../config-optional/remove-ingressgateway-service.yml)
+   - Enable the `metrics-server` by setting `metric_proxy.enable_metrics_server: true` in <cf_install_values_path>.
```

Comment:
Any consideration to making an option in generate-values for this?  it could be more scriptable that way, rather than having to hand edit the file after the fact

Reply:
We hadn't thought about it but could take a look

Reply:
added `--enable-metrics-server` or `-m` flag

Reply:
Heads up @zrob and @heycait, we're generally working towards reducing the amount of code/logic inside the generate-values script as we're aiming to one day get rid of it entirely.